src/ProjectNeon/Assets/Features/Effects/Evade.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+public class Evade : Effect
+{
+    private Member _performer;
+    private Target _effectTarget;
+
+    public void Apply(Member source, Target target)
+    {
+        _performer = source;
```

Comment:
Why are we storing these member fields, when they could be directly passed to the evaluation method?

Reply:
@SilasReinagel Didn't get it. `Evade` effect is triggered on an `Attack`, so we need to keep these fields to know who performed the `Evade` buff (who added this evade effect to the stack) and who is being benefited by it

Reply:
This issue is still open.

What part is confusing? We don't need any fields in the class.

Reply:
@paulodamaso Forgot to tag you.

Reply:
@SilasReinagel `Evade` works as any other triggered `Effect`: first, we apply it, then we execute it when triggered. If someone have the `Evade` buff, the evade effect is active on him (preferrably using the decorators that limit events by turns or by repetitions). The `Evade` effects is triggered before an attack happens, so we need to check if we are applying it to the correct target. So we need to save the original target received on `Apply`